src/main/java/jalau/at18/azul/Tiles.java

```diff
@@ -0,0 +1,20 @@
+package jalau.at18.azul;
+
+public enum Tiles {
```

Comment:
Not sure if this enum was already added by @DaniMB95 ? you can try to rebase and see if this class is not added as new in your PR. This has a correction to be `Tile` instead of `Tiles`

Reply:
deleting duplicate Enum Class for Tiles